Dockerfile.amd64

```diff
@@ -1,5 +1,19 @@
+FROM alpine
+RUN mkdir -p /tmp/dikastes
+RUN chmod 0777 /tmp/dikastes
+
 FROM scratch
 
 ADD bin/dikastes-amd64 /dikastes
 ADD bin/healthz-amd64 /healthz
-CMD ["/dikastes"]
\ No newline at end of file
+# Typical Linux systems start numbering human users at 1000, reserving 1-999
+# for services, so we pick 999 to be least likely to overlap.  It's not a big
+# deal if we happen to overlap, as it would take a container escape for
+# an attacker to use any privileges that happen to be assigned to this user.
+#
+# Precreate the /var/run/dikastes directory so that we don't need any elevated
+# permission to create the directory at runtime.
+COPY --chown=999 --from=0 /tmp/dikastes /var/run/dikastes
```

Comment:
I think this would be clearer if you named the FROM alpine as `FROM alpine AS dev-env` and then the copy could be `COPY --chown=999 --from=dev-env ....`.
Or at least from my reading I think that would work.

Reply:
Yeah, can do